### PR TITLE
Remove non java-gradle-plugin customPlugin snippet

### PR DESCRIPTION
**Pull request by jvandort**
All tags which were previously defined in this snippet were moved elsewhere.
Moved tests from this snippet to the java-gradle-plugin snippet which previously
did not define any tests

### CODE REVIEW

**Comment by jvandort** [comment]
@bot-gradle test BD

**Comment by jvandort** [comment]
@bot-gradle test BD

File: subprojects/docs/src/snippets/base/customExternalTask/groovy/task/build.gradle [hunk 1]
```diff
@@ -6,17 +6,23 @@ plugins {
...
 }
 
-dependencies {
-    testImplementation 'junit:junit:4.13'
+testing {
+    suites {
+        test {
+            useJUnit()
+        }
+    }
```
Comment (big-guy) [inline subprojects/docs/src/snippets/base/customExternalTask/groovy/task/build.gradle:25]: Unfortunately, I don't think we can do this quite yet. This is an incubating feature and we need to show the stable version for now.

File: subprojects/docs/src/snippets/base/customExternalTask/kotlin/task/build.gradle.kts [hunk 2]
```diff
@@ -6,17 +6,23 @@ plugins {
...
+// end::gradle-api-dependencies[]
 // end::external-task-build[]
 
 repositories {
     mavenCentral()
 }
 
-dependencies {
-    testImplementation("junit:junit:4.13")
+testing {
```
Comment (big-guy) [inline subprojects/docs/src/snippets/base/customExternalTask/kotlin/task/build.gradle.kts:20]: Same thing here

**Review by big-guy — CHANGES_REQUESTED** [review]

**Comment by jvandort** [comment]
@bot-gradle test BD

**Review by big-guy — APPROVED** [review]

**Comment by jvandort** [comment]
@bot-gradle test and merge

**Comment by jvandort** [comment]
@bot-gradle test and merge

**Pull request merged**